cluster.go

```diff
@@ -435,6 +421,10 @@ This might be due to one or several causes:
 	}
 
 	// Cluster is ready.
+
+	// Bootstrap the DHT now that we possibly have some connections
```

Comment:
When consensus becomes ready is because Raft has chosen a leader (therefore it has bootstrapped if it needs to bootstrap). 

This may not be the case for crdts but we do trigger another round of bootstrap during `Join` so I think things might work as they are (we'll have to test).